api/src/main/java/io/opentelemetry/api/metrics/MeterProvider.java

```diff
@@ -43,5 +44,5 @@ static MeterProvider getDefault() {
    * @param instrumentationVersion The version of the instrumentation library.
    * @return a tracer instance.
    */
-  Meter get(String instrumentationName, String instrumentationVersion);
+  Meter get(String instrumentationName, @Nullable String instrumentationVersion);
```

Comment:
@MariusVolkhart Thanks for this change - the implementation looks good, but I agree that we don't need to add the `@Nullable` annotation to parameters that, we defensively check, but still expect to not be `null`. Do you think you can remove the annotations?

Reply:
Done! :)